.eslintrc.js

```diff
@@ -30,6 +31,7 @@ module.exports = {
 	rules: {
 		'consistent-return': 'off',
 		'curly': 'error',
+		'import/extensions': ['error', 'ignorePackages'],
```

Comment:
The documentation is not very clear to me on this, but why could this not be `always`?

Reply:
imports from 3rd party modules should mapped using an [import-map](https://github.com/WICG/import-maps) instead of hardcoding filesystem paths that depend on the particular package manager being used

Reply:
i mean leaflet is dep free right now so I guess `always` makes sense too.. but trying to lint extensions in the first place seems funny to me, since it's not really about the extension so much as whether the file can be fetched from an http server. bare specifiers should also work when accompanied by an import-map for the reason described above.

Reply:
if the intent is to be able to see if native module loading actually works, having at least one example in debug load natively still seems like a more accurate way to start to me. but yeah, unless I'm missing something I'm pretty sure we would need some http server for that to work (zero attachment to serve, I like ecstatic for dev stuff although it's no longer maintained)

Reply:
The intent is to have native module loading work on the distribution first. Aligning towards ESM, such as having correct file extensions is already a step forward.